config/config.go

```diff
@@ -358,6 +371,50 @@ func parseDatabaseURL(cfg *Prest) {
 	}
 }
 
+// fetchJWKS tries to get the JWKS from the URL in the config
+func fetchJWKS(cfg *Prest) {
+	if cfg.JWTWellKnown == "" {
+		log.Debugln("no JWT WellKnown url found, skipping")
+		return
+	}
+	if cfg.JWTJWKS != "" {
+		log.Debugln("JWKS already set, skipping")
+		return
+	}
+
+	// Call provider to obtain .well-known config
+	r, err := http.Get(cfg.JWTWellKnown)
+	if err != nil {
+		log.Errorf("Cannot get .well-known configuration from '%s'. err: %v\n", cfg.JWTWellKnown, err)
+		return
+	}
+	defer r.Body.Close()
+
+	var wellKnown map[string]interface{}
+	err = json.NewDecoder(r.Body).Decode(&wellKnown)
+	if err != nil {
+		log.Errorf("Failed to decode JSON: %v\n", err)
+		return
+	}
+
+	//Retrieve the JWKS from the endpoint
+	JWKSet, err := jwk.Fetch(context.Background(), wellKnown["jwks_uri"].(string))
```

Comment:
can you split this conversion into a variable? just to make sure that it never panics:

```go
uri, ok := wellKnown["jwks_uri"].(string)
...
JWKSet, err := jwk.Fetch(context.Background(),uri)
```

Reply:
Done in latest commit.
https://github.com/prest/prest/pull/871/commits/fee30e7ae1162e49e1d88f7fa384c73b016fdc95